.copywrite.hcl

```diff
@@ -0,0 +1,27 @@
+project {
+  # Leave blank if you don't wish to license the project
+  # Default: "MPL-2.0"
+  license = "MPL-2.0"
+
+  # (OPTIONAL) Represents the year that the project initially began
+  # Default: <the year the repo was first created>
+  copyright_year = 2013
+
+  # (OPTIONAL) A list of globs that should not have copyright or license headers .
+  # Supports doublestar glob patterns for more flexibility in defining which
+  # files or folders should be ignored
+  # Default: []
+  header_ignore = [
+    "*.hcl2spec.go",
+    "hcl2template/testdata/**",
+    "test/**",
+    "**/test-fixtures/**",
+    "examples/**",
+    "hcl2template/fixtures/**",
+    "website/**",
+    ".release/**",
+    ".vagrant/**",
+    ".github/**",
+    ".golangci",
```

Comment:
I think these ones can receive headers. Though the `.vagrant` is probably there for local dev, yeah? I see it on the `.gitignore`, which makes sense - that one is a good one to stay

Reply:
Yeah I guess it can't hurt for them to receive headers, I figured CI configs weren't really something we wanted to copyright, but at the same time I'm not against it.
I'll do a second pass on those. The `.vagrant` one I thought was being tracked, but if that's purely local, we probably shouldn't add it to that file, so I'll remove it also.

Come to think of it you're right, most of those can receive copyrights, the only one I think we can keep is the `website` one, since most of the files that are copyrightable are coming from other teams or are just managing dependencies.

Thanks for pointing those out!